src/segmentmytif/features.py

```diff
@@ -25,42 +30,112 @@ def from_string(s):
             raise ValueError()
 
 
-def get_features(input_data: np.ndarray, input_path: Path, feature_type: FeatureType, features_path: Path, profile,
-                 **extractor_kwargs):
-    """
-    Extract features from the input data, or load them from disk if they have already been extracted.
+def get_features(
+    raster: xr.DataArray,
+    input_path: Path,
+    feature_type: FeatureType,
+    features_path: Path,
+    chunk_overlap: int = 200,
```

Comment:
```suggestion
    chunk_overlap: int = 16,
```

Reply:
See artifacts in this segmentation sample:
![image](https://github.com/user-attachments/assets/b9c18455-c9b4-43f0-ba8f-5b5a15fd24d9)
Edge effects are indeed present up to about 16 pixels from the edge. However, it seems to be more like 17 or 18 pixels wide. I don't understand how the effects can be present after more than 16 pixels but it clearly still shows. Maybe we should choose a slightly larger overlap. 20 or 25 or something to be sure.

Reply:
I changed this to 25